BLUESPAWN-client/src/hunt/hunts/HuntT1037.cpp

```diff
@@ -24,22 +24,22 @@ namespace Hunts{
 
 		if(level == Aggressiveness::Cursory || level == Aggressiveness::Normal) {
 			if (file.GetFileAttribs().extension == L".exe" && !bFileSigned) {
-				reaction.FileIdentified(std::make_shared<FILE_DETECTION>(file.GetFilePath()));
+				reaction.FileIdentified(std::make_shared<FILE_DETECTION>(file));
 				return 1;
 			}
 			if(!result && result.vKnownBadRules.size() > 0) {
-				reaction.FileIdentified(std::make_shared<FILE_DETECTION>(file.GetFilePath()));
+				reaction.FileIdentified(std::make_shared<FILE_DETECTION>(file));
 				return 1;
 			}
 		}
 		if(level == Aggressiveness::Normal) {
 			if((std::find(sus_exts.begin(), sus_exts.end(), file.GetFileAttribs().extension) != sus_exts.end())) {
 				LOG_INFO(L"Startup with suspicious extension identified.");
-				reaction.FileIdentified(std::make_shared<FILE_DETECTION>(file.GetFilePath()));
+				reaction.FileIdentified(std::make_shared<FILE_DETECTION>(file));
```

Comment:
LNK files always exist on the in the start folders. We need to find a way to filter

Reply:
I want to add a way to analyze the target of a lnk file. we actually have a few yara rules that scan for suspicious stuff in lnk files, but that's still not that good and obviously not being used here